src/components/select/select.js

```diff
@@ -202,13 +203,19 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $par
       var valueEl = element.find('md-select-value');
       var isReadonly = angular.isDefined(attr.readonly);
 
+      mdSelectCtrl.mdSelectHeaderInput = element.find('md-select-header').find('input')[0];
```

Comment:
So the select header is designed to be an input? Wouldn't it make more sense to extract the complete `md-select-header` element?